src/System.Private.CoreLib/shared/System/SpanHelpers.Byte.cs

```diff
@@ -199,10 +200,22 @@ public static unsafe int IndexOf(ref byte searchSpace, byte value, int length)
             IntPtr index = (IntPtr)0; // Use IntPtr for arithmetic to avoid unnecessary 64->32->64 truncations
```

Comment:
Do we know why this code is currently using `IntPtr` rather than the:
```csharp
#if BIT64
using nint = System.Int64;
#else
using nint = System.Int32;
#endif
```
code that everywhere else seems to prefer?

Reply:
This is left-over from times when this lived in CoreFX and it was not compiled bitness-specific. It would be good for readability to switch this over to nuint.

Reply:
Is a bit messy to clean that up in this PR, will do a follow up